File: problema 3/fortran/problema_3.f90
program suma_promedio
    implicit none
    integer :: numero(10), i, suma = 0
    real :: promedio = 0.0

    do  i = 1, 10
        print *, "Ingrese un numero: "
        read *, numero(i)
        suma = suma + numero(i)
        
    end do
    print *, "Suma de los numeros ingresados: ", suma
    
    promedio = suma / 10.0
    
    print *, "Promedio de los numeros ingresados: ", promedio
    
end program suma_promedio

File: problema 3/c-languaje/problema_3.c
#include <stdio.h>

int main() {
    int numero[10],i, suma = 0;
    float promedio = 0;

    //calculando la suma de los arrays
    for ( i = 0; i < 10; i++)
    {
        printf("Ingrese un numero: ");
        scanf("%d", &numero[i]);

        suma += numero[i];
    }

    printf("Suma de los numeros ingresados: %d\n", suma);

    //calculando el promedio
    promedio = suma/10.00;
    printf("Promedio de los numeros ingresados: %.2f\n", promedio);
    return 0;
}